soundmoneysafemode/README.md

```diff
@@ -0,0 +1,8 @@
+**Monero: Sound Money, Safe Mode** is the August 2020 update of the Monero Means Money lecture/documentary.  Dr. Daniel Kim provides a comprehensive overview of Monero and Bitcoin within the larger context of gold and fiat currency in the 2020 financial crisis.  The source footage for this presentation was livestreamed August 7-8, 2020 as the keynote talks for Monero Village at DEF CON 28.  The streamlined and remastered video, which the subtitles in this repository apply to, premiered September 9, 2020 on the Monero Community Workgroup Youtube channel, and is freely viewable at [Youtube](https://www.youtube.com/watch?v=aC9Uu5BUxII).  
+
+## Translations
+
+Please review the translations [tips for Monero translators](https://github.com/monero-ecosystem/monero-translations/blob/master/translation-tips.md). And also make use of the glossaries availabe [here](https://translate.getmonero.org/dictionaries/monero/) and [here](https://github.com/monero-ecosystem/monero-translations/tree/master/terminology-guides) to keep consistency along with other localized resources.
+
+#### Follow this instructions to translate the subtitles to your native language:
+"README.md" 73 lines, 4443 characters
```

Comment:
Is this intended?

Reply:
This was a copy of README.md from Monero Means Money, with the top section rewritten to describe Sound Money Safe Mode.  So I didn't review the rest of the README.md, it might be out of date.

Reply:
oops, found my error, will resubmit